api/controller/auth.js

```diff
@@ -61,21 +96,106 @@ const signup = async (req, res) => {
   }
 };
 
+const createProfile = async (req, res) => {
+  try {
+    const {
+      email,
+      name,
+      gender,
+      age,
+      country,
+      phoneNumber,
+      countryCode,
+      description,
+      socialMediaLink,
+      profilePicture,
+    } = req.body;
+
+    if (!email) {
+      return res.status(400).json({ message: "Please Enter Email" });
+    }
+
+    if (
+      (!name,
+      !gender,
+      !age,
+      !country,
+      !phoneNumber,
+      !countryCode,
+      !description,
+      !socialMediaLink,
+      !profilePicture)
+    ) {
+      return res.status(400).json({ message: "Please Fill All Fields" });
+    }
```

Comment:
Can we do this validation using express-validator middleware ?? Same for all validations?
https://express-validator.github.io/docs/

Reply:
ok i will add express-validator validations